{"name": "# C5125 Change password and verify the message is displayed: \"Old and new password cannot be same.\"", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \n\n", "trace": "  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python39\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python39\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\C5125_Change_password.py\", line 8, in go_to_chng_pswrd_pg\n    context.app.main_page.go_to_chng_pswrd_pg()\n  File \"C:\\Everything\\IT\\Testing\\Automation_08_09_2019\\UIautomation_python\\pages\\main_page.py\", line 390, in go_to_chng_pswrd_pg\n    wait.until(EC.element_to_be_clickable(USR_NM)).click()\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python39\\lib\\site-packages\\selenium\\webdriver\\support\\wait.py\", line 80, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "steps": [{"name": "Given Loginpage", "status": "passed", "start": 1606886567740, "stop": 1606886573641}, {"name": "Then Login with the given credentials", "status": "passed", "start": 1606886573642, "stop": 1606886576657}, {"name": "Then Go to the \"Change Password\" page https://devcloud.connectedio.com/profile/change-password", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \n\n", "trace": "  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python39\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python39\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\C5125_Change_password.py\", line 8, in go_to_chng_pswrd_pg\n    context.app.main_page.go_to_chng_pswrd_pg()\n  File \"C:\\Everything\\IT\\Testing\\Automation_08_09_2019\\UIautomation_python\\pages\\main_page.py\", line 390, in go_to_chng_pswrd_pg\n    wait.until(EC.element_to_be_clickable(USR_NM)).click()\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python39\\lib\\site-packages\\selenium\\webdriver\\support\\wait.py\", line 80, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "start": 1606886576657, "stop": 1606886600012}, {"name": "And https://devcloud.connectedio.com/profile/change-password is open", "status": "skipped", "start": 1606886602089, "stop": 1606886602089}, {"name": "Then Enter the old password in the field \"New Password\" MyUSA2016!@", "status": "skipped", "start": 1606886602089, "stop": 1606886602089}, {"name": "And Enter the old password in the field \"Confirm New Password\" MyUSA2016!@", "status": "skipped", "start": 1606886602089, "stop": 1606886602089}, {"name": "Then Click on the button \"Save\"", "status": "skipped", "start": 1606886602089, "stop": 1606886602089}, {"name": "And Verify the words Old and new password cannot be same. is on the page", "status": "skipped", "start": 1606886602089, "stop": 1606886602089}], "start": 1606886562575, "stop": 1606886602089, "uuid": "c4c6ce4b-731c-46b8-b216-598195c58332", "historyId": "a5d11e5ff03f07b28c9cdee589ef29b6", "fullName": "features/tests/Login page/C5125_Change_password.feature:# C5125 Change password and verify the message is displayed: \"Old and new password cannot be same.\"", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "# C5125 Change_password"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}